core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties

```
@@ -0,0 +1,24 @@
+TokenDisplayedOnce=Copiez ce jeton maintenant, il ne pourra pas \u00eatre retrouv\u00e9 ult\u00e9rieurement.
+AddNewToken=Ajouter un jeton
+GenerateNewToken=G\u00e9n\u00e9rer
+RevokeToken=R\u00e9voquer ce jeton
+CopyToken=Copier ce jeton
+NoTokenYet=Il n''y a pas de jeton enregistr\u00e9 pour cet utilisateur.
+TokenLastUse=Utilis\u00e9 <b>{0}</b> fois, le dernier usage date de <b>{1}</b> jours
+StatisticsDisabled=Pas de statistique disponible
+StatisticsDisabled.Title=Les statistiques d''utilisation du jeton sont d\u00e9sactiv\u00e9es
+TokenNeverUsed=Jamais utilis\u00e9
+TokenNeverUsedTooltip=Il n''y a pas de date de derni\u00e8re utilisation pour ce jeton
+TokenNeverUsed.Title=Il est fortement recommand\u00e9 de supprimer les jetons qui ne sont pas utilis\u00e9s
+ConfirmRevokeSingle=Confirmez-vous la suppression de ce jeton? Les applications l''utilisant ne pourront plus se \
```

```suggestion
ConfirmRevokeSingle=Confirmez-vous la suppression de ce jeton ? Les applications l''utilisant ne pourront plus se \
```

Interrogation mark in french must have a space before contrary to the english ones.